ca/server.go

```diff
@@ -510,7 +518,11 @@ func (s *Server) Run(ctx context.Context) error {
 			case api.EventDeleteNode:
 				rootReconciler.DeleteNode(v.Node)
 			}
-
+		case <-externalTLSCredsChange:
+			s.signingMu.Lock()
+			s.externalCA.UpdateTLSConfig(NewExternalCATLSConfig(
+				s.securityConfig.ClientTLSCreds.Config().Certificates, s.externalCAPool))
```

Comment:
Sorry for the delay in response!  Yes, it is eventually consistent - during that period, signing nodes may fail, but in a recoverable way, so eventually the correct certs will be issued.  I've added a comment to that effect.